dataset.py

```diff
@@ -100,31 +100,6 @@ def get_batch(self):
         return balanced_batch_images, balanced_batch_texts
 
 
-def hierarchical_dataset(root, opt, select_data='/'):
-    """ select_data='/' contains all sub-directory of root directory """
-    dataset_list = []
-    dataset_log = f'dataset_root:    {root}\t dataset: {select_data[0]}'
-    print(dataset_log)
```

Comment:
Remember to remove print statement in production code

Reply:
Removed print statements!